docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+services:
+  node: 
+    build: .
+    restart: always
+    ports: 
+      - 3000:3000
+    depends_on: 
+      - db
+  db:
```

Comment:
Which port is the db service exposing?

Reply:
default: 5432

Reply:
May I set it as I did in previous commits:
ports:
      - "5000:5432"

Reply:
but setting it that way I found that It doesn't give me right message when it is running
<img width="648" alt="Screen Shot 2019-12-05 at 16 44 43" src="https://user-images.githubusercontent.com/52409462/70245344-961ca800-177e-11ea-8395-3cbeca0266a1.png">